src/Parser/Warrior/Arms/Modules/Abilities.js

```diff
@@ -1,24 +1,23 @@
 import SPELLS from 'common/SPELLS';
 
 import CoreAbilities from 'Parser/Core/Modules/Abilities';
+import ISSUE_IMPORTANCE from 'Parser/Core/ISSUE_IMPORTANCE';
 
 class Abilities extends CoreAbilities {
   spellbook() {
     return [
```

Comment:
It looks like this module is slightly outdated. I'd recommend defining the combatant locally here. This will be useful when you need to check if a talent or tier bonus is active.
```
spellbook() {
  const combatant = this.combatants.selected; //add this line
  return [
```